### DISCUSSION TITLE
IOS APP新版本1.3.3，资料库“立即同步”功能故障

### DISCUSSION THREAD

**Discussion by lf2cjj** [navidrome/navidrome#3485] [General]
RT，
环境：NAS-docker安装Navidrome Music Server，版本0.51.0，IOS手机安装音流APP，版本1.3.3
1.开启离线模式——点击“设置--资料库--立即同步”，没有任何反馈和交互。（未更新到NAS上的新数据，同步失败）
2.关闭离线模式——点击“设置--资料库--立即同步”，没有任何反馈和交互。（未更新到NAS上的新数据，同步失败）
注1：以上操作中，偶尔会有 点击“立即同步”时，按钮文案变成“取消同步”的交互。大多时候不会
注2：以上操作在移动网络和wifi环境下都尝试过多次，都未能成功同步资料库

PS：上个版本，同步资料库，有一行文案“同步于n分钟之前”这个交互，在这个版本没有了。同步体验降低。

**Comment by lf2cjj** [comment]
@deluan

**Comment by deluan** [comment]
> RT, Environment: Navidrome Music Server installed on NAS-docker, version 0.51.0, Audio Stream APP installed on IOS phone, version 1.3.3 1. Turn on offline mode - click "Settings--Library--Synchronize Now", no feedback or interaction. (No new data updated on NAS, synchronization failed) 2. Turn off offline mode - click "Settings--Library--Synchronize Now", no feedback or interaction. (No new data updated on NAS, synchronization failed) Note 1: In the above operations, occasionally when clicking "Synchronize Now", the button text changes to "Cancel Synchronization". Most of the time it won't Note 2: The above operations have been tried many times in mobile network and wifi environments, but they have not been able to successfully synchronize the database
> 
> PS: In the previous version, when synchronizing the database, there was a line of text "Synchronize n minutes ago". This interaction is gone in this version. The synchronization experience is degraded.

You are describing options in the StreamMusic app, not in Navidrome. Please post your issue/question in https://github.com/gitbobobo/StreamMusic/issues
